modules/home/assets/js/components/home-screen.js

```diff
@@ -1,5 +1,4 @@
 import { Container, Box, Stack } from '@elementor/ui';
-
 import TopSection from './top-section';
```

Comment:
I don't think that the current PR is working.

I think that this line needs to be updated first: 
`<GetStarted getStartedData={ props.homeScreenData.get_started[ 0 ] } />`

Also I believe we need to make this conditional.